packages/react-vapor/src/components/drop/tests/DropPod.spec.tsx

```diff
@@ -162,27 +162,59 @@ describe('DropPod', () => {
+
+                        expect(portalSpy).toHaveBeenCalledWith('🍟', divElement);
+                    });
+
+                    it('should append the div element to the Defaults.DROP_ROOT as the portal root if no selector is passed in props', () => {
+                        const divElement = document.createElement('div');
                         const expectedElement = document.querySelector('head');
                         Defaults.DROP_ROOT = 'head';
 
                         jest.spyOn(document, 'querySelector');
-                        const portalSpy = jest.spyOn(ReactDOM, 'createPortal');
+                        const appendChildSpy = jest.spyOn(expectedElement, 'appendChild');
                         shallow(<DropPod renderDrop={() => '🍟'} ref={buttonRef} />, {}).dive();
 
-                        expect(portalSpy).toHaveBeenCalledWith('🍟', expectedElement);
+                        expect(appendChildSpy).toHaveBeenCalledWith(divElement);
                     });
 
-                    it('should create a portal with the selector prop if passed to the dropPod', () => {
+                    it('should append the div element to with the selector prop as the portal root if passed to the dropPod', () => {
+                        const divElement = document.createElement('div');
                         const expectedElement = document.querySelector('head');
                         Defaults.DROP_ROOT = '#🥔';
 
                         jest.spyOn(document, 'querySelector');
-                        const portalSpy = jest.spyOn(ReactDOM, 'createPortal');
+                        const appendChildSpy = jest.spyOn(expectedElement, 'appendChild');
+
                         shallow(<DropPod renderDrop={() => '🍟'} selector="head" ref={buttonRef} />, {}).dive();
 
-                        expect(portalSpy).toHaveBeenCalledWith('🍟', expectedElement);
+                        expect(appendChildSpy).toHaveBeenCalledWith(divElement);
                     });
+
+                    it('should remove the div element with the portal root', () => {
+                        const divElement = document.createElement('div');
+                        const expectedElement = document.querySelector('head');
+                        Defaults.DROP_ROOT = 'head';
+
+                        jest.spyOn(document, 'querySelector');
+                        const removeChildSpy = jest.spyOn(expectedElement, 'removeChild');
+                        const componentToUnmount = shallow(
+                            <DropPod renderDrop={() => '🍟'} ref={buttonRef} />,
+                            {}
+                        ).dive();
+
+                        componentToUnmount.unmount();
+
+                        expect(removeChildSpy).toHaveBeenCalledWith(divElement);
+                    });
+
+                    // expect(ReactDOM.createPortal).toHaveBeenCalledWith('🍟', expectedElement);

```

Comment:
was this forgotten? :yum:

Reply:
@sallainCoveo should I remove the comment or remove the line?

Reply:
I think so, I don't remember what this line was supposed to be used for.